README.md

```diff
@@ -14,18 +15,21 @@ Follow the [installation instructions](https://docs.datadoghq.com/serverless/ins
 
 We're excited to share that our next-generation, low-overhead Lambda Extension is now generally available. This new extension is packaged alongside our existing extension, yet offers substantially faster cold starts as well as lower resource consumption.
 
-You can opt out of the next-generation Lambda Extension by setting the the environment variable `DD_EXTENSION_VERSION: compatibility` and using extension version `v67` or later.
+You can opt out of the next-generation Lambda Extension by setting the environment variable `DD_EXTENSION_VERSION: compatibility` and using extension version `v67` or later.
 
 ### Supported Configurations
+
 Today, all workloads using Logs and Metrics are supported.
 
 APM Tracing is supported for Python, NodeJS, Go, Java, and .NET runtimes.
```

Comment:
Should we merge and say 

`Today, all workloads using Logs, Metrics, and Traces are supported for runtimes...` And remove line 22?

Just curious if we somehow changes this to be Python and NodeJS only for APM tracing and then we added the other runtimes.